client/app/components/composites/ListingCard/ListingCard.js

```diff
@@ -55,10 +55,9 @@ class ListingCard extends Component {
     const distanceFormatted = formatDistance(listing.distance, localeCode);
     const price = listing.price;
     const moneyFormatted = price ? formatMoney(price.get(':money'), localeCode) : null;
-    const pricingUnitTranslations = price && price.get(':pricingUnit').size > 0 ?
-      price.get(':pricingUnit') :
-      null;
-    const pricingUnit = `/ ${localizedString(pricingUnitTranslations, 'pricing unit')}`;
+    const hasPricingUnit = price && price.get(':pricingUnit') != null;
```

Comment:
The format of unknown pricing unit is not yet defined in DiscoveryAPI, therefore we check the existence of key (':pricingUnit') and possible null value.